os-eap7-launch/added/launch/activemq-subsystem.xml

```diff
@@ -15,13 +14,10 @@
             <param name="direct-deliver" value="false"/>
         </http-acceptor>
         <in-vm-acceptor name="in-vm" server-id="0"/>
-        <broadcast-group name="bg-group1" connectors="http-connector" jgroups-channel="ee"/>
-        <discovery-group name="dg-group1" jgroups-channel="ee"/>
-        <cluster-connection name="my-cluster" address="jms" connector-name="http-connector" discovery-group="dg-group1"/>
         <jms-queue name="ExpiryQueue" entries="java:/jms/queue/ExpiryQueue"/>
         <jms-queue name="DLQ" entries="java:/jms/queue/DLQ"/>
         <!-- ##DESTINATIONS## -->
         <connection-factory name="InVmConnectionFactory" connectors="in-vm" entries="java:/ConnectionFactory"/>
-        <connection-factory name="RemoteConnectionFactory" ha="true" block-on-acknowledge="true" reconnect-attempts="-1" connectors="http-connector" entries="java:jboss/exported/jms/RemoteConnectionFactory"/>
+        <connection-factory name="RemoteConnectionFactory" connectors="http-connector" entries="java:jboss/exported/jms/RemoteConnectionFactory" reconnect-attempts="-1" />
```

Comment:
Should it still be reconnect-attempts="-1" for non-ha?  (as this option isn't being set in non-ha version of standalone-full.xml )

Reply:
@jmesnil Is the non-HA default of econnect-attempts -1?

Reply:
yes, keeping -1 (infinite reconnection) is correct.
By default, in non-HA case, Artemis clients never tries to reconnect (reconnect-attempts = 0).
In cloud environment, it is expected that the Artemis servers should always be up.
It might not be the case during deployment rolling upgrades of the Artemis servers but keeping reconnect-attempts to -1 ensures that the Artemis clients will "wait" until the Artemis servers are up again.